Source/Scene/Model.js

```diff
@@ -2839,6 +2919,27 @@ define([
         };
     }
 
+    function createColorFunction(model) {
+        return function() {
+            return model.color;
+        };
+    }
+
+    function createColorBlendFunction(model) {
```

Comment:
Does 3D Tiles also use this?  If so, should we make this a utility function on `ColorBlendMode` that takes a `blendAmount`?